plasma_framework/contracts/src/exits/utils/ExitId.sol

```diff
@@ -2,31 +2,32 @@ pragma solidity 0.5.11;
 
 import "../../utils/Bits.sol";
 import "../../utils/PosLib.sol";
+import "./OutputId.sol";
 
 library ExitId {
     using PosLib for PosLib.Position;
-    using Bits for uint160;
+    using Bits for uint168;
     using Bits for uint256;
 
+    uint8 constant internal FIRST_BIT = 167;
```

Comment:
I guess this can go completely private as it is a hidden logic abstracted by the API already
consider rename this as `FIRST_BIT_LOCATION` instead. When looking at shifting I have always thought 256 - FIRST_BIT is like 256 - 1.